extra-utils/pastebinit/spec

```diff
@@ -1,4 +1,4 @@
 VER=1.5
 SRCTBL="https://launchpad.net/pastebinit/trunk/$VER/+download/pastebinit-$VER.tar.bz2"
```

Comment:
You should switch to the new ACBS [spec format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/). For your reference, here's a rewrite of your `spec` file...

```
VER=1.5
REL=4
SRCS="tbl::https://launchpad.net/pastebinit/trunk/$VER/+download/pastebinit-$VER.tar.bz2"
CHKSUMS="sha256::42e5a84ce7e46825fb3b6478e11893fad357197327257e474bd0d3549f438457"
```